src/Pages/Welcome/welcome.jsx

```diff
@@ -1,12 +1,21 @@
 import { useState } from "react";
 import Wave from "../../Assets/koala-hug.gif";
+import { TextArea } from "../../Components/Input";
 
 export default () => {
-  const [name, setName] = useState(localStorage.getItem("UserName") || "There");
+  const [typeText, setTypeText] = useState("");
+
```

Comment:
```suggestion
```

Reply:
Fixed in 925d48b0a98fb7bd42aa97c5a80d13fa92be9888